internal/npm_package/packager.js

```diff
@@ -34,13 +38,22 @@ function write(p, content) {
 // overwrite the version in package.json)
 
 function main(args) {
-  const [outDir, baseDir, srcsArg, binDir, genDir, depsArg, packPath, publishPath] = args;
+  const [outDir, baseDir, srcsArg, binDir, genDir, depsArg, replacementsArg, packPath, publishPath] = args;
 
+  const replacements = [
+    // Strip content between BEGIN-INTERNAL / END-INTERNAL comments
+    [/(#|\/\/)\s+BEGIN-INTERNAL[\w\W]+END-INTERNAL/g, ''],
+  ];
+  const rawReplacements = JSON.parse(replacementsArg);
+  for (let key of Object.keys(rawReplacements)) {
```

Comment:
```
for (const key in rawReplacements)
```
is a little cleaner and avoids Object.keys

Reply:
We discourage that pattern in the TypeScript style guide since you can pick up prototype properties by mistake